virttest/utils_misc.py

```diff
@@ -2105,19 +2105,27 @@ def get_cpu_vendor(cpu_info="", verbose=True):
     return vendor
 
 
-def get_support_machine_type(qemu_binary="/usr/libexec/qemu-kvm"):
+def get_support_machine_type(qemu_binary="/usr/libexec/qemu-kvm", remove_alias=False):
     """
-    Get the machine type the host support,return a list of machine type
+    Get the machine type the host support,return a tuple (s, c, v),
+    s, c, v are list, such as:
+    'pc', 'RHEL 7.4.0 PC (i440FX + PIIX, 1996)', '(alias of pc-i440fx-rhel7.4.0)'
```

Comment:
Your example is no very clear here, `(['some', 'thing'], ['another thing'],['type1', 'type2'])`

Reply:
@hereischen  thanks, updated to make the annotations more clearly, could you help review again ?